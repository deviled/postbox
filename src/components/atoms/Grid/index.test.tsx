import React from 'react';
import { render } from '__tests__/utils';
import { FlexBox, FlexCol } from '.';

describe('Grid.tsx', () => {
  it('should match FlexBox snapshot', () => {
    const { baseElement } = render(<FlexBox mb={4} mt={4} bgcolor="background" />);

    expect(baseElement).toMatchSnapshot();
  });

  it('should match FlexCol snapshot', () => {
    const { baseElement } = render(<FlexCol mr={4} ml={4} bgcolor="error" />);

    expect(baseElement).toMatchSnapshot();
  });
});
