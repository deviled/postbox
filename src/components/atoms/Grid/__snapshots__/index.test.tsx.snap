// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid.tsx should match FlexBox snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  background-color: #efefef;
  margin-bottom: 4px;
  margin-top: 4px;
}

<body>
  <div>
    <div
      class="c0"
    />
  </div>
</body>
`;

exports[`Grid.tsx should match FlexCol snapshot 1`] = `
<body>
  .c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  background-color: #f44336;
  margin-right: 4px;
  margin-left: 4px;
}

.c1 {
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div>
    <div
      class="c0 c1"
    />
  </div>
</body>
`;
